qm.spec.rpkg

```diff
@@ -58,7 +58,7 @@ sed -i '/user_namespace/d' qm.if
 %install
 # install policy modules
 %_format MODULES $x.pp.bz2
-%{__make} DESTDIR=%{buildroot} DATADIR=%{_datadir} install install-setup install-quadlet
+%{__make} DESTDIR=%{buildroot} DATADIR=%{_datadir} install
```

Comment:
We could remove the `DATADIR=%{_datadir}` as well since the defaults are the same as that expected by Fedora. But no biggie either way.